src/SocialAuthUserManager.php

```diff
@@ -156,20 +167,22 @@ public function setSessionKeysToNullify(array $session_keys) {
    *   The social implementer identifier.
    * @param string $provider_user_id
    *   The unique id returned by the user.
+   * @param JSON $data
```

Comment:
JSON is not a PHP data type. 
$data is probably either a (serialized) string or an array. Don't exactly know how this work yet